### DISCUSSION TITLE
Welcome to HyperTS Discussions!

### DISCUSSION THREAD

**Discussion by zhangxjohn** [DataCanvasIO/HyperTS#43] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by python888888** [comment]
安装了dask库，总是报找不到这个库的错，为什么呢

**Reply by zhangxjohn** [reply]
您好，可以详细描述一下您的问题吗？我们目前没有发现过在安装过程中存在关于dask无法发现的错误。
